extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -202,6 +212,11 @@
               "title": "Campaign ID",
               "type": "integer",
               "description": "Identifier of the marketing campaign to which activity originating this message belongs to."
+            },
+            "xdm:campaignName": {
```

Comment:
Same discussion here for `xdm:campaignName` as part of the _campaign_ schema/object

Reply:
https://github.com/adobe/xdm/pull/472#discussion_r213328077